.github/workflows/prs.yml

```diff
@@ -68,7 +68,7 @@ jobs:
       - name: Run unit tests
         run: |
           cd src/backup-metadata/
-          go test -v ./internal/...
+          go test -v ./internal/... ./test/binary/...
```

Comment:
Wouldn't it be easier to just do something like `go test -v ./...`? Would future-proof it too if we add more packages or whatever

Reply:
The problem is there is a `test/e2e` package in here that are integration tests which expect a full cluster / `velero` binary to run against it.

https://github.com/cloudfoundry/capi-k8s-release/tree/master/src/backup-metadata/test/e2e

I see your point, though, maybe I can separate those out.

Reply:
Ah dang, well this is probably fine for now then. If we start adding packages in here, then maybe we revisit then